core/theme.js

```diff
@@ -60,18 +60,26 @@ Blockly.Theme = function(name, opt_blockStyles, opt_categoryStyles,
 
   /**
    * The font style.
-   * @type {Blockly.Theme.FontStyle}
+   * @type {!Blockly.Theme.FontStyle}
    * @package
    */
   this.fontStyle = /** @type {Blockly.Theme.FontStyle} */ (Object.create(null));
 
   /**
    * Whether or not to add a 'hat' on top of all blocks with no previous or
    * output connections.
-   * @type {boolean}
+   * @type {?boolean}
    * @package
    */
-  this.startHats = false;
+  this.startHats = null;
+
+  /**
+   * The accessibility style.
+   * @type {!Blockly.Theme.AccessibilityStyle}
+   * @package
+   */
+  this.accessibilityStyle =
```

Comment:
Same idea as the above comment.

Reply:
In which case maybe it can live in componentStyle then.